src/elements/public/Admin/Log/ErrorEntry.ts

```diff
@@ -0,0 +1,444 @@
+              class="p-m bg-base shadow-xs rounded-t-l rounded-b-l"
+              layout="horizontal"
+              >
+          </foxy-spinner>
+        </div>
+      `;
+    } else {
+      return html`
+        <div aria-busy=${this.in('busy')}
+             aria-live='polite'
+             class='${this.nohide? 'nohide': ''} ${ this.data.hide_error ? 'fadeout': ''} flex flex-auto content-center w-full ${this.data.hide_error ? 'text-tertiary': ''}'>
+          <vaadin-checkbox
+            title="Hide this error"
+            class="pt-s"
+            ?disabled=${!this.in('idle')}
+            .value=${this.data._links.self.href}
+            ?checked=${this.data.hide_error}
+            @change=${this.__handleCheckErrorEntry}
+          ></vaadin-checkbox>
+          <article class='m-s text-body'>
+            <header class='border-l-2 p-s relative ${this.open ? 'border-error' : 'border-primary'}'>
+              <vaadin-button @click=${this.__toggleOpen}
+                    theme="icon"
+                    class='text-s absolute right-s top-s rounded-full bg-transparent top-0 m-0 p-0'
+                    aria-label='toggle'
+                    >
+                    ${this.open
+                        ? html`<iron-icon icon='icons:expand-less'></iron-icon>`
+                        : html`<iron-icon icon='icons:expand-more'></iron-icon>`
+                        }
+              </vaadin-button>
+              ${this.data.hide_error ? html`<span class="px-s py-xs text-s font-medium tracking-wide rounded bg-success-10 text-success">Hidden</span>` : ''}
+              <foxy-i18n key='date' options='{"value": "${this.data.date_created}"}' class='text-s ${this.open ? 'text-error' : 'text-primary'}'></foxy-i18n>
+              <foxy-i18n key='time' options='{"value": "${this.data.date_created}"}' class='text-s ${this.open ? 'text-error' : 'text-primary'}'></foxy-i18n>
+              <div>${this.data.error_message}</div>
+            </header>
+            ${this.open ? html`
+                  <main>
+                    ${(this.data._links as any)['fx:customer']?.href
+                        ? html`
+                          <foxy-custom-box title="Customer">
+                            <foxy-customer-info href="${(this.data._links as any)['fx:customer']?.href}"></foxy-customer-info>
+                          </foxy-custom-box>
+                        `
+                        : ''
+                        }
+                        ${(this.data._links as any)['fx:transaction']?.href
+                            ? html`
+                              <foxy-custom-box title="Transaction">
+                                <foxy-transaction-info href="${(this.data._links as any)['fx:transaction']?.href}"></foxy-transaction-info>
+                              </foxy-custom-box>
+                            `
+                            : ''
+                        }
+                        ${this.data.user_agent}
```

Comment:
**issue:** user agent is displayed twice

The one at the top of the expanded card needs to be removed.

Reply:
092031035b19c4a94afaf24958ec81bb09d0c4d0
Thanks!